BannedScript/banScript.py

```diff
@@ -0,0 +1,60 @@
+import yaml
+import pymongo
+import requests
+
+# Load YAML file
+with open('./config/config.yml', 'r') as file:
+    config = yaml.safe_load(file)
+
+authToken = input("Please provide your token: ")
+username = input("Please enter username to whom you want to ban: ")
+
+dbUrl = config['database']['url']
```

Comment:
why are constants not capitalized? these are basically constants that we are taking from the config file